frontend/app/src/services/rotkehlchen-api.ts

```diff
@@ -872,6 +872,22 @@ export class RotkehlchenApi {
       .then(handleResponse);
   }
 
+  queryBinanceMarkets(): Promise<string[]> {
+    return this.axios
+      .get<ActionResult<string[]>>('/exchanges/binance/pairs')
+      .then(handleResponse);
+  }
+
+  queryBinanceUserMarkets(name: string, location: string): Promise<string[]> {
+    return this.axios
+      .get<ActionResult<string[]>>('/exchanges/binance/pairs/' + name, {
```

Comment:
this is a nitpick but would it be easy to change that to a template format instead? 

https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html